docs/usage/DirectMigrationExampleAPI.md

```diff
@@ -0,0 +1,200 @@
+#Direct Migration using API
+
+Please setup, deploy and configure the Migration Operator as well as the Migration Controller
+appropriately before proceeding further.
+
+We will be performing a migration of a Django Application consisting of persistent volumes and image streams, this
+application is already present on the source cluster used in the following migration example.
+
+The steps to perform a Direct Migration using Migration Toolkit for Containers (MTC) are as follows:
+
+**1. Add Migration clusters**<br>
+This step involves addition and configuration of the source as well as the destination clusters 
+involved in the migration.
+
+- At first, let's create a MigCluster instance in order to add the destination cluster. 
```

Comment:
We should use bullet points when we have a list of things, but not for a single piece of information.

Reply:
Done, updated the PR.